### DISCUSSION TITLE
Link Genes to 'active motifs'

### DISCUSSION THREAD

**Discussion by BartBryant** [stuart-lab/signac#1719] [Q&A]
first, I Luv Signac and am grateful for the maintenance of this awesome ATAC pkg!!! 

Question:  
After obtaining 'active motifs' (by chromVar), how can i link genes to a 'active motif' of interest? 
More specifically, 
based on my ATAC data that found transcription factor 'A' highly active in cell type 1 over cell type 2, 
how can i obtain a gene list responsible for higher motif activity for transcription factor 'A' in celltype1 over celltype 2?

Sorry if my query has already been addressed, 
again, luv Signac and its upkeep by Tim!!!

**Comment by timoast** [comment]
Hi @BartBryant, thanks for the kind words!

This is an interesting question, we don't currently have any examples of how to do this in the Signac vignettes but you might want to check out some tools like SCENIC+: https://www.nature.com/articles/s41592-023-01938-4. These methods aim to construct the gene regulatory network taking into account the peak accessibility, presence of TF motifs, and TF gene expression. Dictys is another approach that might be useful: https://www.nature.com/articles/s41592-023-01971-3

**Comment by BartBryant** [comment]
Hi @timoast , 
thanks for links to other tools, will defly check 'em out!!!

With ChatGPT v4's help, was able to write code (in R) to obtain a gene list associated with any motif of interest. The code uses fxns from ChIPseeker along with other relevant pkgs. This initial code will help us start to link 'motif activity' to specific genes sets
AND
with the other pkgs you mentioned above, I will be able to learn some dope biology related to my research topic.
 
The pkgs mentioned above use Python. Currently i am quasi-familiar with R. To learn Python I am wrangling various resources such as 'Python Crash Course' by Eric Matthes. 
If you have other resources you can pass along to help expedite my code-learning journey that would be uber dopE, grAci.

best and thx again for developing Signac 
AND 
for keeping it maintained (other pkgs not maintained creates bifurcations from my main biological-based goals)
barT
